configs/common/packages.yaml

```diff
@@ -143,6 +145,8 @@
       # also: ... extdata2g segfault UFS?
       version: ['2.40.3']
       variants: ~shared ~extdata2g ~pflogger
+      #version: ['2.46.2']
+      #variants: +pflogger
```

Comment:
We may want to retain the variants for 2.40.3 (`~shared ~extdata2g ~pflogger`), but since this is just a comment it won't hurt. We can do a test install with the variants in line 147 on one platform first, have the UFS developers test it, and then roll it out or correct as necessary.